### PR TITLE
fix link to blog

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
I'm not entirely sure that this is the intended link (v. say
`https://daml.com/blog`), but the current one is dead so presumably this
is better.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: docs/source/support/releases.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ Release notes for each release are published on the `Release Notes section of th
 Roadmap
 -------
 
-Once a month Digital Asset publishes a community update to accompany the announcement of the release candidate for the next release. The community update contains a section outlining the next priorities for development. You can find community updates on the `Daml Driven Blog <https://daml.com/daml-driven>`_, or subscribe to the mailing list or social media profiles on `https://daml.com/ <https://daml.com/>`_ to stay up to date.
+Once a month Digital Asset publishes a community update to accompany the announcement of the release candidate for the next release. The community update contains a section outlining the next priorities for development. You can find community updates on the `Daml Driven Blog <https://daml.com/blog/engineering>`_, or subscribe to the mailing list or social media profiles on `https://daml.com/ <https://daml.com/>`_ to stay up to date.
```
Comment (cocreature) [inline docs/source/support/releases.rst:52]:
```suggestion
Once a month Digital Asset publishes a community update to accompany the announcement of the release candidate for the next release. The community update contains a section outlining the next priorities for development. You can find community updates on the `Daml Driven Blog <https://daml.com/blog>`_, or subscribe to the mailing list or social media profiles on `https://daml.com/ <https://daml.com/>`_ to stay up to date.
```

Maybe link to the general blog instead? I guess here both work so your choice.

**Review by cocreature — APPROVED** [review]
Thanks!

File: docs/source/support/releases.rst (hunk 1 above)
Comment (garyverhaegen-da) [reply docs/source/support/releases.rst:52]: It's much easier to find the mentioned monthly community updates on the engineering tab at the moment.

Comment (cocreature) [reply docs/source/support/releases.rst:52]: alright then let’s go with that.

**Pull request merged**